snappass/main.py

```diff
@@ -22,13 +24,29 @@
 }
 
 
+def check_redis_alive(fn):
+    def inner(*args, **kwargs):
+        try:
+            redis_client.ping()
```

Comment:
Note that with this `ping()` design, it is still possible (even if, admittedly, unlikely), that the ping works but the function call below still fails to a `ConnectionError`.

Why not simply wrap the function call`fn(*args, **kwargs)` in the `try/except` ?